.circleci/config.yml

```diff
@@ -711,6 +721,25 @@ jobs:
       - persist_to_workspace: *artifacts_executables
       - gitter_notify_failure_unless_pr
 
+  b_ubu_static_arm:
+    <<: *base_ubuntu2004_arm_large
+    environment:
+      TERM: xterm
+      MAKEFLAGS: -j 5
+      CMAKE_OPTIONS: -DCMAKE_BUILD_TYPE=Release -DUSE_Z3_DLOPEN=ON -DSTRICT_Z3_VERSION=OFF -DUSE_CVC4=OFF -DSOLC_STATIC_STDLIBS=ON
+    steps:
+      - checkout
+      - run:
+          name: "install dependencies"
+          command: sudo apt-get --quiet --assume-yes --no-install-recommends install libboost-all-dev z3 libz3-dev
```

Comment:
@cameel can you elaborate how these packages are built and pushed to the PPA?

Reply:
The script I linked builds a `.deb` source package and then uploads to [`~ethereum/cpp-build-deps` PPA](https://launchpad.net/~ethereum/+archive/ubuntu/cpp-build-deps) on Launchpad. We run it locally whenever a new version of Z3 is released.

To be honest I'm only vaguely familiar with it. I know we run the script when we update Z3 but so far I never had to do it myself and I don't know much about how it's set up. In particular not sure if you could fit an ARM version into the same PPA or if we'd need a separate one.

Actually, I see that the source package has `Architecture: any` so it's possible that it might be as simple as selecting ARM from some list when the source package is uploaded. I remember that when @Marenz last ran the other PPA script (for solc) he had to choose an Ubuntu version. @leo Do you know how it works?

Reply:
I was initially compelled to simply change `Architecture: any` but in this case the source code must be compiled in the target architecture, then packed and uploaded. Meaning if I change and someone runs the script for ARM from an X86 the library files won't work.

As for the PPA, like most modern artifact storage, it can receive multiple packages from multiple architectures. Launchpad won't have any problem receiving packages for ARM.